### DISCUSSION TITLE
希望能做个看评论区楼层的功能，谢谢。爱发电已经赞助了。

### DISCUSSION THREAD

**Discussion by Towards-future** [the1812/Bilibili-Evolved#3019] [功能建议 / Ideas]
希望能做个看评论区楼层的功能，谢谢。爱发电已经赞助了。如果要过多的赞助我可以继续给。联系方式3491900649@qq.com

**Comment by Geeyun-JY3** [comment]
可以用这个脚本：<https://greasyfork.org/scripts/403348>，是兼容的

另外重复：#473

**Comment by Towards-future** [comment]
很抱歉。因为就在今天，那个翻页评论区失效了。B站今天又搞鬼。据说昨天B站老版本APP看不到评论区了。然后今天我自己试这个脚本，就失效了。昨天晚上还好好的。




------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2022年2月17日(星期四) 晚上7:05
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [the1812/Bilibili-Evolved] 希望能做个看评论区楼层的功能，谢谢。爱发电已经赞助了。 (Discussion #3019)





 
可以用这个脚本：https://greasyfork.org/scripts/403348，是兼容的
 
另外重复：#473
 
—
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Reply by Geeyun-JY3** [reply]
才发现……

**Comment by the1812** [comment]
无能为力. 我以前尝试做过一个, 然后 API 就被改了失效了

**Comment by Towards-future** [comment]
好吧，依然感谢你做的Bilibili-Evolved，还是给我们带来了很好的使用体验




------------------&nbsp;原始邮件&nbsp;------------------
发件人: "Grant ***@***.***&gt;; 
发送时间: 2022年2月17日(星期四) 晚上7:34
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [the1812/Bilibili-Evolved] 希望能做个看评论区楼层的功能，谢谢。爱发电已经赞助了。 (Discussion #3019)





 
无能为力. 我以前尝试做过一个, 然后 API 就被改了失效了
 
—
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.
